contentcuration/contentcuration/frontend/channelEdit/components/edit/EditView.vue

```diff
@@ -178,7 +186,12 @@
         return this.$tr('editingMultipleCount', totals);
       },
       areDetailsValid() {
-        return !this.oneSelected || this.getContentNodeDetailsAreValid(this.nodeIds[0]);
+        // only run this if the modal isn't closing
+        // because async DOM updates during close can cause issues
+        if (!this.closingModal) {
```

Comment:
In EditModal, closingModal starts out as `false` at initialization, and the only line that sets its value in there also sets it to `false`, so I don't think this guard is having any effect.

If it is working fine at the moment, I'd recommend just removing all the `closingModal` references and thus simplify the diff.